smithy-model/src/main/java/software/amazon/smithy/model/traits/EnumTrait.java

```diff
@@ -28,19 +28,23 @@
 /**
  * Constrains string values to one of the predefined enum constants.
  */
-public final class EnumTrait extends AbstractTrait implements ToSmithyBuilder<EnumTrait> {
+public class EnumTrait extends AbstractTrait implements ToSmithyBuilder<EnumTrait> {
     public static final ShapeId ID = ShapeId.from("smithy.api#enum");
 
-    private final List<EnumDefinition> definitions;
+    protected final List<EnumDefinition> definitions;
```

Comment:
Why protected? I think protected properties should be extremely rare and hidden behind, at most, a protected method to access the property. Bonus points would be to make sure this class can't be extended outside this package if possible.

Reply:
That was a holdover from a point in time where the synthetic trait was doing more complicated things. It has been fixed.